manifest.yml

```diff
@@ -32,6 +32,7 @@ applications:
       EASEY_DB_MAX_USES_BEFORE_CONN_RECREATE: ((maxUsesBeforeRecreatingConnection))
       EASEY_DB_SQL_LOGGING: ((sqlLogging))
       EASEY_DB_MAX_QUERY_EXECUTION_TIMEOUT: ((maxQueryExecutionTime))
+      EASEY_MONITOR_PLAN_API_ENABLE_AUDIT_LOG: true
```

Comment:
If we're going to add the `enableAuditLog` variables, I think this should use it here. However, since they're all set to `true`, then I think that variable can be removed from the `manifest-vars.*.yml` files.